src/hash_mergers/answer.rb

```diff
@@ -9,33 +9,19 @@ class Answer < HashMerger
       private
 
       def hash_array(*a)
-        default_array(*a).concat super
+        super.unshift(domain_answers)
       end
 
-      def default_array(groups:, node:)
-        [default_hash(:domain)].tap do |x|
-          x.push(default_hash(:group)) if groups
-          if node == 'local'
-            x.push(default_hash(:local))
-          elsif node
-            x.push(default_hash(:node))
-          end
-        end
-      end
-
-      def default_hash(section)
-        section_default(section).each_with_object({}) do |(key, value), memo|
-          memo[key] = value.default unless value.default.nil?
-        end
+      def domain_answers
```

Comment:
I'm not sure I follow this method name - possibly I'm mistaken, but aren't these the 'defaults' rather than the `domain_answers`?

Reply:
Opps good job picking that up. It's been renamed.